### ISSUE TITLE
Annex A of OGC Moving Features Encoding Extension: netCDF Best Practice

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/movingfeatures#3]
Clause 2 of [OGC Moving Features Encoding Extension: netCDF (OGC 16-114r3)](http://docs.opengeospatial.org/bp/16-114r3/16-114r3.html) says "Conformance with this specification shall be checked using all the relevant tests specified in Annex A (**normative**)."

However, the header of Annex A says "Annex A: Conformance Test (**informative**)".

So Clause 2 says Annex A is normative, but the header of Annex A says the annex is informative.

**Comment by ghobona** [comment]
Cc: @desruisseaux

**Comment by desruisseaux** [comment]
Indeed the text is unclear. The normative part is that the file shall be conform to CF-Convention (also OGC 11-162r2). The informative part is that one way to verify that is to try to open the file using the UCAR library.

If we edit this document, maybe it would be better to convert it to Asciidoc first?

**Comment by ghobona** [comment]
@desruisseaux Yes, please use [metanorma-asciidoc](https://www.metanorma.org/author/ogc/).
